Cannot find module 'fs-extra'

```
[ember 02/14/17 12:47:31] Cannot find module 'fs-extra'
[ember 02/14/17 12:47:31] Error: Cannot find module 'fs-extra'
[ember 02/14/17 12:47:31]     at Function.Module._resolveFilename (module.js:326:15)
[ember 02/14/17 12:47:31]     at Function.Module._load (module.js:277:25)
[ember 02/14/17 12:47:31]     at Module.require (module.js:354:17)
[ember 02/14/17 12:47:31]     at require (internal/module.js:12:17)
[ember 02/14/17 12:47:31]     at Object.<anonymous> (/export/home/tester/hudson/data/workspace/MP_TRUNKDEV_POSTCOMMIT/crt-frontend_0f29aca5408b1bc71d2b93d2e222c4796e8501a4/crt-frontend-ember/node_modules/ember-engines/node_modules/broccoli-dependency-funnel/node_modules/mr-dep-walk/lib/deps-from-file.js:2:12)
[ember 02/14/17 12:47:31]     at Module._compile (module.js:410:26)
[ember 02/14/17 12:47:31]     at Object.Module._extensions..js (module.js:417:10)
[ember 02/14/17 12:47:31]     at Module.load (module.js:344:32)
[ember 02/14/17 12:47:31]     at Function.Module._load (module.js:301:12)
[ember 02/14/17 12:47:31]     at Module.require (module.js:354:17)
[ember 02/14/17 12:47:31]     at require (internal/module.js:12:17)
[gradle] ------- process completed.
:crt-frontend-ember:emberBuild FAILED
```

Neither `mr-dep-walk` nor `broccoli-dependency-funnel` define `fs-extra` as a dependency. The latter defines it as a dev dependency.

--- a/package.json
+++ b/package.json
@@ -1,7 +1,8 @@
 {
   "dependencies": {
     "acorn": "^4.0.4",
-    "amd-name-resolver": "^0.0.6"
+    "amd-name-resolver": "^0.0.6",
+    "fs-extra": "^2.0.0"
   },
   "devDependencies": {
     "chai": "^3.5.0",

